web-registry/src/components/organisms/ChooseCreditClass/ChooseClassTemplate.tsx

```diff
@@ -0,0 +1,70 @@
+import React from 'react';
```

Comment:
The single folder pattern is useful to split code that is meant to only work together but here it seems that it's 2 different reusable components.
We have a call with @victordelval tomorrow to start clarifying this so there's no more confusion about this.

Reply:
They are different components, but they are both part of the same MUI Grid (`container` and `item`), so they are meant to work together. This is a grey-area case. If 2 style components depend on each other, should they be considered reusable? I'm open to it - maybe it is best to separate, and add a comment about how it is not meant to be resuable?

>ChooseClassTemplate should probably be moved under template folder 

Same kind of thought here. The name Template is maybe misleading, because the styling and component mix is intended only for this use-case. Maybe there's a better name, like `ChooseClassScaffold` or `ChooseClassWrapper`

Reply:
The spirit of this component is: DOM structure and style. 
It is intended to be reusable just for the ChooseClass (legacy and regular)

Reply:
As far as organization, my intent is to keep these together as they are same-feature-related.

Reply:
Did some separating and reorganizing. I can update more if needed.

Reply:
Thanks for clarifying this @mhagel, and also for the splitting you've already done. If this code is meant to only work together let's keep it that way then.

Regarding naming, one convention I find relevant is to have the main component of the folder (entry point) with the same name as the folder. Another is to use dot notation to indicate that a component is used as children to another. We could also use the semantic grid/item for naming. Here it seems there are 2 entry points (normal and legacy) so it could be maybe:

`ChooseCreditClass`
`ChooseCreditClassLegacy`
`ChooseCreditClass.Grid` (prev `ChooseClassTemplate`)
`ChooseCreditClass.Item` (prev `ClassOption`)

What do you think of this?

Reply:
Note that this is just for discussion it doesn't necessarily need to be changed in this PR (especially since conventions are still being debated). Just wanted to have your thoughts on that.

Reply:
I ended up using these suggestions. I think these are good instincts. Having contextual naming is very useful for other devs. They feel a bit long and wordy, but they are easier to search for, so ultimately it is probably better to repeat the "ChooseCreditClass" prefix.

Reply:
that seems very good to me!